conf/config_handler_mock.go

```diff
@@ -0,0 +1,35 @@
+//go:build unit
+
+package conf
+
+import (
+	"github.com/aerospike/aerospike-management-lib/asconfig"
+	"github.com/go-logr/logr"
+)
+
+// TODO: Use a gomock instead. It is more robust in its assertions.
+
+type mockCFG struct {
+	valid            bool
+	err              error
+	validationErrors []*asconfig.ValidationErr
+	confMap          *asconfig.Conf
+	confText         string
+	flatConf         *asconfig.Conf
+}
+
+func (o *mockCFG) IsValid(log logr.Logger, version string) (bool, []*asconfig.ValidationErr, error) {
+	return o.valid, o.validationErrors, o.err
+}
+
+func (o *mockCFG) ToMap() *asconfig.Conf {
+	return o.confMap
+}
+
+func (o *mockCFG) ToConfFile() asconfig.DotConf {
+	return o.confText
+}
+
+func (o *mockCFG) GetFlatMap() *asconfig.Conf {
+	return o.flatConf
+}
```

Comment:
I don't think we should include mock files here. They could go in a mocks sub package like in https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/aerospikereceiver/cluster/mocks

Reply:
not needed